serlib/ser_notation_gram.ml

```diff
@@ -37,9 +37,11 @@ type grammar_constr_prod_item =
   [%import: Notation_gram.grammar_constr_prod_item]
   [@@deriving sexp]
 
-type level =
-  [%import: Notation_gram.level]
-  [@@deriving sexp]
+module Notation = struct
+  type level =
+    [%import: Notation.level]
+    [@@deriving sexp]
+end
```

Comment:
This needs to go in `Ser_notation`, done in the main branch.

Reply:
Yes, as I wrote above, I wanted to put it there but that caused an error with cyclic dependencies. I will look in the main branch to see what you did.

Reply:
Oh the dependency among `Constrexpr` and `Notation` was just inverted upstream.